.travis.yml

```diff
@@ -4,7 +4,7 @@ language: python
 matrix:
   include:
     - python: "3.6-dev"
-      env: TEST_CMD="flake8"
```

Comment:
I think I saw a separate PR for this recently?

Reply:
Not that I know of. Jelle's PR was about upgrading the version of flake8. But I'll move this to the config where it belongs better.